testplan/exporters/testing/pdf/__init__.py

```diff
@@ -198,14 +199,17 @@ def create_pdf(self, source):
                 reportlab_styles.extend(row_data.style)
 
         pdf_path = self.cfg.pdf_path
+        pathlib.Path(pdf_path).resolve().parent.mkdir(
+            parents=True, exist_ok=True
+        )
         try:
             if self.cfg.interactive_port is None:
                 override = True
             else:
                 override = False
         except AttributeError:
             override = True
-        if not override and os.path.exists(pdf_path):
+        if not override and pathlib.Path(pdf_path).exists():
             file_name, ext = os.path.splitext(pdf_path)
```

Comment:
you can use pathlib.Path here as well, looks better than mixing two path library